dozent/dozent.py

```diff
@@ -58,36 +77,42 @@ def _make_date_from_date_link(date_link: Dict[str, str]) -> datetime.date:
+        day = int(date_link["day"]) if date_link["day"] != "NaN" else 1
+        month = int(date_link["month"])
+        year = int(date_link["year"])
 
         return datetime.date(year=year, month=month, day=day)
 
-    def get_links_for_days(self, start_date: datetime.date, end_date: datetime.date) -> List:
+    def get_links_for_days(
+        self, start_date: datetime.date, end_date: datetime.date
+    ) -> List:
         """
         Function to get the links for the given start and end days
         :return: date dictionaries that are within self.start_date and self.end_dates
         """
         links = []
 
         if not FIRST_DAY_OF_SUPPORT >= start_date:
-            RuntimeError(f'We currently only support the range {FIRST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}, '
-                         f'{LAST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}'
-                         f'\nWe\'re planning on adding support for it soon.'
-                         f'Need that data sooner? Add an issue to out GitHub repo and we\'ll '
-                         f'walk you through the process'
-                         f'Link to our GitHub '
-                         f'Issues: https://github.com/Twitter-Public-Analysis/Twitter-Public-Analysis/issues')
+            RuntimeError(
+                f'We currently only support the range {FIRST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}, '
+                f'{LAST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}'
+                f"\nWe're planning on adding support for it soon."
+                f"Need that data sooner? Add an issue to out GitHub repo and we'll "
+                f"walk you through the process"
+                f"Link to our GitHub "
+                f"Issues: https://github.com/Twitter-Public-Analysis/Twitter-Public-Analysis/issues"
+            )
 
         elif not end_date <= end_date:
-            RuntimeError(f'We currently only support the range {FIRST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}, '
-                         f'{LAST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}'
-                         f'\nWe\'re planning on adding support for it soon.'
-                         f'Need that data sooner? Add an issue to out GitHub repo and we\'ll '
-                         f'walk you through the process'
-                         f'Link to our GitHub '
-                         f'Issues: https://github.com/Twitter-Public-Analysis/Twitter-Public-Analysis/issues')
+            RuntimeError(
+                f'We currently only support the range {FIRST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}, '
+                f'{LAST_DAY_OF_SUPPORT.strftime("%d, %b %Y")}'
+                f"\nWe're planning on adding support for it soon."
+                f"Need that data sooner? Add an issue to out GitHub repo and we'll "
+                f"walk you through the process"
+                f"Link to our GitHub "
+                f"Issues: https://github.com/Twitter-Public-Analysis/Twitter-Public-Analysis/issues"
+            )
```

Comment:
Are these formatting changes from black or did you do them on your own?

Reply:
Definitely Black conforming to PEP8

Reply:
Alright, I think it looks good!